src/librustc_infer/infer/region_constraints/mod.rs

```diff
@@ -784,7 +784,6 @@ impl<'tcx> RegionConstraintCollector<'_, 'tcx> {
         )
     }
 
-    /// See [`RegionInference::region_constraints_added_in_snapshot`].
     pub fn region_constraints_added_in_snapshot(&self, mark: &Snapshot<'tcx>) -> Option<bool> {
```

Comment:
Probably it was meant to refer to https://doc.rust-lang.org/nightly/nightly-rustc/rustc_infer/infer/struct.InferCtxt.html#method.region_constraints_added_in_snapshot -- maybe you can fix it instead of removing it?

Reply:
Was getting `intra_doc_link_resolution_failure` after trying a few paths, so decided to just put it in code block